packages/atomic/src/components/result-template-components/atomic-result-value/atomic-result-value.tsx

```diff
@@ -19,15 +28,32 @@ export class AtomicResultValue {
    */
   @Prop() value!: string;
   /**
-   * Which highlight should the value be highlighted with
+   * Which highlight should the value be highlighted with.
+   * Possible values are `firstSentencesHighlights`, `excerptHighlights`, `printableUriHighlights` or `summaryHighlights`.
+   *
    */
   @Prop() shouldHighlightWith?: string;
+  /**
+   * The format in which to display the result value.
+   * Possible values are `text`, `date`, `number` or `currency`.
+   */
+  @Prop() format: ResultValueFormat = 'text';
+  /**
+   * When the `format` is `date`, defines the format of the date itself.
+   * Available formats: https://day.js.org/docs/en/display/format
+   */
+  @Prop() dateFormat = 'D/M/YYYY';
+  /**
+   * When the `format` is `number`, allows to define the number of digits.
+   */
+  @Prop() numberOfDigits?: number;
+  /**
+   * When the `format` is `currency`, allows to define which currency to display.
+   */
+  @Prop() currency = 'USD';
```

Comment:
E-commerce sites typically display prices in the end-user's currency. Aside from the currency code, the number needs to be converted using the exchange rate. How would one achieve this with Atomic?